api.js

```diff
@@ -469,5 +469,10 @@ module.exports = function(_roommanager, storage) {
 		});
 	});
 
+	/* Spotify Authentication Callback url */
+	router.get("/spotify/authentication", (req, res) => {
+		res.redirect(`${req.query.state}?${req.query.code}`);
```

Comment:
@dyc3 not really sure how to send the data with the websocket

Reply:
You can use the session from the request (`req.session`) to compare it with clients connected to rooms (`roommanager.js`)